src/core/datastore/processors/cache-offline-data-processor.js

```diff
@@ -62,6 +63,7 @@ export class CacheOfflineDataProcessor extends OfflineDataProcessor {
 
   _processCreate(collection, data, options) {
     let offlineEntity;
+    data = clone(data);
```

Comment:
Looks odd. Are you sure you meant this?

Reply:
This is to avoid mutating the objects that are passed by the user. What do you find odd?